baseapp/abci.go

```diff
@@ -368,7 +415,7 @@ func handleQueryStore(app *BaseApp, path []string, req abci.RequestQuery) abci.R
 	return resp
 }
 
-func handleQueryP2P(app *BaseApp, path []string, _ abci.RequestQuery) (res abci.ResponseQuery) {
+func handleQueryP2P(app *BaseApp, path []string, _ abci.RequestQuery) abci.ResponseQuery {
```

Comment:
We can remove the unused param
```suggestion
func handleQueryP2P(app *BaseApp, path []string) abci.ResponseQuery {
```